jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/IO.java

```diff
@@ -559,6 +559,25 @@ public static long write(GatheringByteChannel out, ByteBuffer[] buffers, int off
 
         return total;
     }
+
+    /**
+     * <p>Convert an object to a {@link File} if possible.</p>
+     * @param fileObject A File, String, Path or null to be converted into a File
+     * @return A File representation of the passed argument or null.
+     */
+    public static File asFile(Object fileObject)
```

Comment:
This doesn't look like a routine we want to generalize.

Reply:
It is called in 2 places and I think there are probably more that could use it.   When users set a configuration as a bean or an attribute, it can be unclear if they are specifying a String filename, File, Path etc. (probably should support URI, URL and Resource as well).  So it is useful to be forgiving of such configuration issues.